source/danger.d.ts

```diff
@@ -323,7 +323,9 @@ interface BitBucketServerChangesValueMove {
   }
 }
 
-type BitBucketServerChangesValue = BitBucketServerChangesValueAddCopyModifyDelete | BitBucketServerChangesValueMove
```

Comment:
Danger complained about this file in this PR, but it looks like this formatting is the only thing that changes.